### PR TITLE
Bug/vspc 207

### PR DESCRIPTION
**Pull request by swetalina-prusty25**
- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

### CODE REVIEW

**Comment by swetalina-prusty25** [comment]
finished the development of an image search field that allows users to search for images by name, file name, and description. It will filter the images.

File: vspace/src/main/java/edu/asu/diging/vspace/core/data/ImageRepository.java [hunk 1]
```diff
@@ -17,6 +17,8 @@
 public interface ImageRepository extends PagingAndSortingRepository<VSImage, String> {
 
     List<VSImage> findByFilenameLikeOrNameLike(String filename, String name);
+    
+    List<VSImage> findByFilenameLikeOrNameLikeOrDescriptionLike(String filename, String name ,String Description);
```
Comment (jdamerow) [inline vspace/src/main/java/edu/asu/diging/vspace/core/data/ImageRepository.java:21]: variables/parameter names are lowercased

Comment (jdamerow) [reply vspace/src/main/java/edu/asu/diging/vspace/core/data/ImageRepository.java:21]: also, blank goes after not before the comma

File: vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ImageService.java [hunk 2]
```diff
@@ -232,6 +232,18 @@ public IVSImage getImageById(String imageId) throws ImageDoesNotExistException {
         results.forEach(r -> imageResults.add(r));
         return imageResults;
     }
+    
+    @Override
+    public List<IVSImage> findByFilenameOrNameContainsOrDescription(String searchTerm) {
```
Comment (jdamerow) [inline vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ImageService.java:237]: just call this method `search`

Comment (jdamerow) [reply vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ImageService.java:237]: also javadoc is missing

File: vspace/src/main/java/edu/asu/diging/vspace/web/staff/api/ImageSearchApiController.java [hunk 3]
```diff
@@ -45,4 +58,32 @@
         }
         return new ResponseEntity<String>(idArray.toString(), HttpStatus.OK);
     }
+    
+    @RequestMapping("/staff/images/searchImagesncd")
```
Comment (jdamerow) [inline vspace/src/main/java/edu/asu/diging/vspace/web/staff/api/ImageSearchApiController.java:62]: let's make this `/staff/images/search/full`

File: vspace/src/main/java/edu/asu/diging/vspace/web/staff/api/ImageSearchApiController.java [hunk 4]
```diff
@@ -45,4 +58,32 @@
...
+    public String searchImageDescription(@RequestParam(value = "searchText", required = false) String searchTerm , Model model, RedirectAttributes attributes) {
```
Comment (jdamerow) [inline vspace/src/main/java/edu/asu/diging/vspace/web/staff/api/ImageSearchApiController.java:63]: this method should be in its own controller

File: vspace/src/main/webapp/WEB-INF/views/staff/images/imagelist.html [hunk 5]
```diff
@@ -199,6 +206,35 @@ <h2>Images</h2>
...
+                	 if($("#searchImageField").val().trim()) {
+                		 
+                          $('#searchImageButton').removeAttr('disabled');
+                	 }
+                	 else {
+                         $('#searchImageButton').attr('disabled','true');
+                	 }
+                	 if (!$('#searchImageButton').attr('disabled') && $("#searchImageField").val().trim()){
+                		 
+                			$("#searchImageLink").attr('href','[[@{/staff/images/searchImagesncd?searchText=}]]'+$("#searchImageField").val());
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/images/imagelist.html:221]: indentation

File: vspace/src/main/webapp/WEB-INF/views/staff/modules/sequences/sequence.html [hunk 6]
```diff
@@ -1,348 +0,0 @@
-<html layout:decorate="~{layouts/main_staff}">
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/modules/sequences/sequence.html:1]: why is this file being deleted?

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
The ticket description is not from the right ticket?

**Pull request closed without merging**